Vanilla/Resources/dictionary.js

```diff
@@ -1,3 +1,4 @@
+// Weather API codes for current weather conditions
```

Comment:
This comment makes it easy for everyone to understand what these keycodes do.

Reply:
Ok, will change